### PR TITLE
*: a vote for tisonkun as TiDB reviewer

### PR DESCRIPTION
**Pull request by bb7133**
As described in the title, thanks.

### CODE REVIEW

File: votes/0546-tisonkun-as-tidb-reviewer.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+I (@bb7133) hereby nominate @tisonkun as TiDB Reviewer and call for a vote.
+
+## Deadline
+
+The vote will be open for at least 2 days unless there is an objection or not enough votes.
+
+## Scope
+
+* TOC
```
Comment (winkyao) [inline votes/0546-tisonkun-as-tidb-reviewer.md:18]: I think the scope should be the TiDB Maintainers?

Comment (tisonkun) [reply votes/0546-tisonkun-as-tidb-reviewer.md:18]: As the template, I'd prefer as “Team TiDB”. After we have a first example, we can refer to it later :)

Comment (bb7133) [reply votes/0546-tisonkun-as-tidb-reviewer.md]: It is updated, PTAL @winkyao @tisonkun

File: votes/0546-tisonkun-as-tidb-reviewer.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+[@tisonkun](https://github.com/tisonkun) has been working on `pingcap/tidb` for several months. IMO he made a lot of great contribution to TiDB, including the tracking of unstable tests(#25899), restructing test cases(#26022). What's more, he shows a great vision to the TiDB community. The details of his contributions are listed as follow:
+
+* [Authored pull requests](https://github.com/pingcap/tidb/commits?author=tisonkun).
+* [Reviewed pull requests](https://github.com/pingcap/tidb/pulls?q=is%3Apr+reviewed-by%3Atisonkun)
+
+I (@bb7133) hereby nominate @tisonkun as TiDB Reviewer and call for a vote.
+
+## Deadline
+
+The vote will be open for at least 2 days unless there is an objection or not enough votes.
```
Comment (tisonkun) [inline votes/0546-tisonkun-as-tidb-reviewer.md:14]:
```suggestion
The vote will be open for at least 3 days unless there is an objection or not enough votes.
```

according to https://github.com/pingcap/community/blob/master/teams/README.md#actions

**Review by wjhuang2016 — APPROVED** [review]

File: teams/tidb/membership.json [hunk 3]
```diff
@@ -92,6 +92,7 @@
         "tsthght",
         "xuyifangreeneyes",
         "zhangjinpeng1987",
-        "zhuo-zhi"
+        "zhuo-zhi",
+        "tisonkun"
```
Comment (disksing) [inline teams/tidb/membership.json:96]: Does it need to sort alphabetically?

Comment (tisonkun) [reply teams/tidb/membership.json:96]: I'd prefer so. cc @bb7133 

BTW @disksing this vote is still under process, not a decision made. You can show your approval by explicitly approving this PR.

Comment (disksing) [reply teams/tidb/membership.json:96]: oh I thought no objection means YES 😅

**Review by disksing — APPROVED** [review]

**Review by cfzjywxk — APPROVED** [review]

**Review by kennytm — APPROVED** [review]

**Comment by bb7133** [comment]
Merge this PR since it has been 3 days, and this vote can be treated accepted by ['Lazy Majority'](https://github.com/pingcap/community/tree/master/teams)

**Pull request merged**